docs/man-openmpi/man3/MPI_Status_get_error.3.rst

```diff
@@ -0,0 +1,71 @@
+.. _mpi_status_get_error:
+
+
+MPI_Status_get_error
+========================
+
+.. include_body
+
+:ref:`MPI_Status_get_error` |mdash| Retrieves the MPI_ERROR field from *status*.
+
+
+SYNTAX
+------
+
+
+C Syntax
+^^^^^^^^
+
+.. code-block:: c
+
+   #include <mpi.h>
+
+   int MPI_Status_get_error(MPI_Status *status, int *error)
+
+
+Fortran Syntax
+^^^^^^^^^^^^^^
+
+.. code-block:: fortran
+
+   USE MPI
+   ! or the older form: INCLUDE 'mpif.h'
+   MPI_STATUS_GET_ERROR(STATUS, ERROR, IERROR)
+   	INTEGER	STATUS(MPI_STATUS_SIZE), IERROR
+   	INTEGER ERROR
+
+
+Fortran 2008 Syntax
+^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: fortran
+
+   USE mpi_f08
+   MPI_Status_get_error(status, error, ierror)
+   	TYPE(MPI_Status), INTENT(INOUT) :: status
```

Comment:
```suggestion
   	TYPE(MPI_Status), INTENT(IN) :: status
```

Reply:
@bosilca thank you, yes, the INOUT is correct for the _set_error version of the functions, but not for the _get_ error. I've updated all _set versions to apply this change